www/index.html

```diff
@@ -4,9 +4,12 @@
     <meta charset="utf-8">
     <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
     <meta http-equiv="Content-Security-Policy"
-        content="default-src 'self' data: gap: https://ssl.gstatic.com http://nominatim.openstreetmap.org https://e-mission.eecs.berkeley.edu 'unsafe-inline';
+        content="default-src 'self' data: gap: https://ssl.gstatic.com http://nominatim.openstreetmap.org https://e-mission.eecs.berkeley.edu https://habitica-assets.s3.amazonaws.com https://www.habitica.com 
+        https://habitica.com/api/v3/members/4f369eef-aed4-4408-bcbf-b34896daf7e3 'unsafe-inline';
```

Comment:
This is necessary, I am using this just to get a members data that does not require authentication. Using this until we have the backend api functionality.